ci-operator/jobs/openshift/frr/openshift-frr-release-main-periodics.yaml

```diff
@@ -0,0 +1,86 @@
+periodics:
+- agent: kubernetes
+  cluster: build05
+  decorate: true
+  extra_refs:
+  - base_ref: release-main
+    org: openshift
+    repo: frr
+  interval: 72h
+  labels:
+    ci-operator.openshift.io/cloud: equinix-ocp-metal
+    ci-operator.openshift.io/cloud-cluster-profile: equinix-ocp-metal
+    ci-operator.openshift.io/cluster: build05
+    ci-operator.openshift.io/variant: periodics
+    ci.openshift.io/generator: prowgen
+    job-release: "4.20"
```

Comment:
this label, is it something we have to manually bump?

Reply:
I think so, as we do in the "main" lane